packages/qwik-city/src/runtime/src/use-endpoint.ts

```diff
@@ -28,7 +28,7 @@ export const loadClientData = async (
   let resolveFn: () => void | undefined;
 
   if (!qData) {
-    const fetchOptions = getFetchOptions(opts?.action);
+    const fetchOptions = getFetchOptions(opts?.action, opts?.clearCache) ?? {};
```

Comment:
Why the {}? No need to create objects here, no? And if it is really needed, better to add it in getFetchOptions

Reply:
Yeah, this is not needed. I remove it.
This is a remnant of a tentative implementation that was modifying opts inplace.